docs/faq/admin.txt

```diff
@@ -109,3 +109,12 @@ Android and iOS.
 
 Depending on feature support, there *may* be minor stylistic differences
 between browsers. These are considered acceptable variations in rendering.
+
+What assistive technologies are supported for using the admin?
+==============================================================
+
+We want the admin to be compatible with wide range of assistive
+technologies, but are aware of many blockers currently. We intend to support
+all latest versions of major assistive technologies, including
+JAWS, NVDA, VoiceOver iOS, TalkBack, Dragon, VoiceOver macOS, Voice Control,
+ZoomText, Windows Contrast Themes, and screen magnifiers.
```

Comment:
Can we somehow avoid "we"?

Reply:
For example: `The admin is intended to be compatible with ...`

Reply:
👍 I can try if we want to, but was also trying hard to avoid passive voice :) I’ll try to think of another sentence.